README.md

```diff
@@ -12,30 +12,37 @@
 
 ## Getting started
 
-LEGO requires `python3.11`, `python3.11-venv`, `docker` and `docker-compose`. Services like Postgres, Redis, Thumbor and Minio run inside docker.
+LEGO requires `python3.11`, `docker`, `docker-compose` and `pdm`. Services like Postgres, Redis, Thumbor and Minio run inside docker. The virtual environment is created by `pdm`.
+
+### Initial setup (only needed once)
 
 ```bash
-# Initial setup (only need to once)
 $ git clone git@github.com:webkom/lego.git && cd lego/
-$ python3 -m venv venv
 $ echo "from .development import *" > lego/settings/local.py
-$ source venv/bin/activate
-$ pip install -r requirements/dev.txt
+$ pdm install
+$ source .venv/bin/activate
```

Comment:
hmm, not a big deal, but I kinda like the non-hidden venv-folder

Reply:
I kinda do as well, but for some reason [`pdm config venv.backend [virtualenv|venv|conda]`](https://pdm.fming.dev/latest/usage/venv/) does not work ...

But, if you already have a `venv`, it will use that instead. So you still get to use that, but just not by default.

Reply:
Updated the docs to clarify this ^